src/flisp/Makefile

```diff
@@ -83,8 +95,19 @@ $(BUILDDIR)/$(EXENAME)-debug: $(DOBJS) $(LIBFILES_debug) $(BUILDDIR)/$(LIBTARGET
 $(BUILDDIR)/$(EXENAME): $(OBJS) $(LIBFILES_release) $(BUILDDIR)/$(LIBTARGET).a $(BUILDDIR)/flmain.o | $(BUILDDIR)/flisp.boot
 	@$(call PRINT_LINK, $(CCLD) $(SHIPFLAGS) $(LDFLAGS) $(OBJS) $(BUILDDIR)/flmain.o -o $@ $(BUILDDIR)/$(LIBTARGET).a $(LIBFILES_release) $(LIBS) $(OSLIBS))
 
-ifneq ($(BUILDDIR),.)
-$(BUILDDIR)/flisp.boot: flisp.boot | $(BUILDDIR)
+$(BUILDDIR)/host/Makefile:
+	mkdir -p $(BUILDDIR)/host
+	@# add Makefiles to the build directories for convenience (pointing back to the source location of each)
```

Comment:
Not the right file source in the comment. Is it worth trying to use the existing code to do this?

Reply:
Fixing the comment. I don't think it's worth trying to deduplicate, esp since this adds a couple extra lines.